src/test/java/de/hs_mannheim/informatik/ct/end_to_end/RoomControllerTest.java

```diff
@@ -190,13 +190,10 @@ public void checkInInvalidCredentials() throws Exception{
     }
 
     @Test
-    public void checkInInvalidRoomPin() throws Exception{
-        // check in with empty username should
+    public void checkInInvalidRoomPin() throws Exception {
         this.mockMvc.perform(
-                post("/r/checkIn")
```

Comment:
This change should be reverted as well, since submitting the PIN in the POST request is a good idea.